ts/src/hitbtc3.ts

```diff
@@ -306,12 +306,261 @@ export default class hitbtc3 extends Exchange {
                 'broad': {},
             },
             'options': {
+                'defaultNetwork': 'ERC20',
+                'defaultNetworks': {
+                    'ETH': 'ETH',
+                    'USDT': 'TRC20',
+                },
                 'networks': {
-                    'ETH': 'USDT20',
-                    'ERC20': 'USDT20',
-                    'TRX': 'USDTRX',
-                    'TRC20': 'USDTRX',
-                    'OMNI': 'USDT',
+                    // mainnet network ids are in lowercase for BTC & ETH
+                    'BTC': 'btc',
+                    'OMNI': 'BTC',
+                    'ETH': 'eth',
+                    'ERC20': 'ETH',
+                    'ETC': 'ETC',
+                    'BEP20': 'BSC',
+                    'TRC20': 'TRX',
+                    'TRX': 'TRX',
+                    // '': 'UGT',
```

Comment:
why these empty comments?

Reply:
all other exchanges had definitions (readable titles) for the network codes, by which i was able to identify and assign them, but in case of hitbtc, either those definitions were not available, or just I have left them for "todo". for now lets leave them and I'll get back to identify them better in future after we merge all important ones.